1_fetch/src/calc_HIT.R

```diff
@@ -0,0 +1,9 @@
+
+
+
+calc_HITmetrics<-function(data,yearType,drainArea,floodThreshold){
+  
+  out_data<-calc_allHIT(x=data,yearType=yearType,drainArea=drainArea,floodThreshold=floodThreshold)
```

Comment:
Let's suppress warnings for the `calc_allHIT` function because all of the warnings are "future warnings" for the developers of the `calc_allHIT` function.
`suppressWarnings(calc_allHIT(x=data,yearType=yearType,drainArea=drainArea,floodThreshold=floodThreshold))`

Reply:
Should be `out_data<-suppressWarnings(calc_allHIT(x=data,yearType=yearType,drainArea=drainArea,floodThreshold=floodThreshold))`

Reply:
ahh - ok, changed this.